docs/README.md

```diff
@@ -1,20 +1,18 @@
 # preCICE #
 
+<p>
 <a style="text-decoration: none" href="https://www.precice.org/" target="_blank">
     <img src="https://img.shields.io/website-up-down-green-red/https/www.precice.org.svg?label=website" alt="preCICE website status">
 </a>
-<a style="text-decoration: none" href="https://travis-ci.org/precice/precice" target="_blank">
-    <img src="https://travis-ci.org/precice/precice.svg?branch=develop" alt="Build status">
-</a>
-<a style="text-decoration: none" href="https://travis-ci.org/precice/systemtests" target="_blank">
-    <img src="https://img.shields.io/travis/precice/systemtests.svg?label=system%20tests&style=flat" alt="Build status">
-</a>
 <a style="text-decoration: none" href="https://github.com/precice/precice/releases/latest" target="_blank">
     <img src="https://img.shields.io/github/release/precice/precice.svg" alt="Release">
 </a>
 <a style="text-decoration: none" href="https://github.com/precice/precice/blob/develop/LICENSE" target="_blank">
```

Comment:
This can actually be removed, as it is static, (kind of) obvious, and can easily be found if needed.